tests/system/cli/test_cli_default_command.py

```
@@ -1,29 +1,34 @@
 TEST_TIMEOUT_SECONDS = 30
+HELP_OUTPUT_COLUMNS = 120
 
 
-def test_cli_help_lists_run_command() -> None:
-    result = subprocess.run(
+def _run_cli_help() -> subprocess.CompletedProcess[str]:
+    help_env = os.environ.copy()
+    help_env["COLUMNS"] = str(HELP_OUTPUT_COLUMNS)
+
+    return subprocess.run(
         ["tunacode", "--help"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)